WTWSMS/events/ACR_celtic_flavor_events.txt

```diff
@@ -348,7 +354,27 @@ character_event = {
 		completely_controls = c_innse_gall
 		
 		NOT = { religion_group = pagan_group }
-		culture_group = celtic
+		OR = {
+			culture_group = celtic
+			culture = romano-british
+			culture = saxon
+			culture = jute
+			culture = angle
+			culture = anglo-saxxon
+			culture = english
+			culture = norman
+			culture = old_saxon
+			culture = briton
+			culture = breton
+			has_landed_title = c_innse_gall
+		}
+
+		NOT = {
+			has_game_rule = {
+				name = supernatural_events
+				value = off
+			}
+		}
```

Comment:
Don't remove the Event. There are plenty of unrealistic Events in this game that can be disabled, such as the immortal quest and the Demon Spawn quest. Some people like this kind of Event, and others do not. Leave it.